### PR TITLE
Update RESPONSIBILITIES.md to disambiguous admin role

### PR DESCRIPTION
**Pull request by seraphjiang**
### Description
Disambiguous admin role in the process. Repo admin is github role, in opensearch-project that means org admin. clarify this to allow community to locate right group people for help if need

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: RESPONSIBILITIES.md
Comment (dblock) [inline RESPONSIBILITIES.md:41]: Folks in the "admin" team aren't org admins. They are "members of the admin team". An org admin has some capabilities such as deleting the org, which none of the folks in the admin team are. Let's rename this to "members of the admin team".

**Review by dblock — CHANGES_REQUESTED** [review]
s/org admin/members of the admin team/g

File: RESPONSIBILITIES.md
Comment (seraphjiang) [reply RESPONSIBILITIES.md]: Thanks @dblock  updated, take a look

**Review by dblock — DISMISSED** [review]

**Comment by seraphjiang** [comment]
@peternied @saratvemulapalli @mch2 looks like we need second reviewer

File: RESPONSIBILITIES.md
Comment (peternied) [inline RESPONSIBILITIES.md:41]: _From https://opensearch.slack.com/archives/C051JEH8MNU/p1686591138909899_

This link isn't visible to the public, looking at a blog post on teams [1], `People who are not members of the organization cannot view any teams.`  Seems like this change doesn't expose this information publicly.

I think we should drop the url since it won't work for anyone that isn't a member of the project, what do you think about coming up with a page like maintainers but for admins that links the team, but also has the list of admins available? 

- [1] https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility

**Comment by dblock** [comment]
> > what's recommended to share this to public, is there a admin.md file somewhere?
> 
> Looks like https://github.com/opensearch-project/.github/blob/main/ADMINS.md could be modified and brought up to date

It's the right idea I think. We don't have to update it in this PR though, @seraphjiang up to you?

**Comment by seraphjiang** [comment]
@dblock @peternied 

Good idea to update https://github.com/opensearch-project/.github/blob/main/ADMINS.md 

The current explanation refer to a github link, more focus on explain the permission, there is need more clarification on org admin, project admin and repo admin and their specific responsibility in opensearch project.

> As an admin you own stewardship of the repository and its settings. Admins have [admin-level permissions on a repository](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization). Use those privileges to serve the community and protect the repository as follows.

**Comment by dblock** [comment]
@peternied How about we merge this PR as is, and we can followup with doing the admin doc refactoring work separately? I know this is a dead link for most people, but it's still something that exists and it's better than what we have now.

**Comment by peternied** [comment]
> I know this is a dead link for most people, but it's still something that exists and it's better than what we have now.

It is better for members of OpenSearch-Project and worse for the broader community.  I would rather be consistent than flaky.   Happy to merge with a link target that anyone can follow.

**Comment by seraphjiang** [comment]
> It is better for members of OpenSearch-Project and worse for the broader community. I would rather be consistent than flaky. Happy to merge with a link target that anyone can follow.

@peternied 
Thanks, when could we expected the link mentioned be ready to incorporate here.

File: RESPONSIBILITIES.md
Comment (peternied) [reply RESPONSIBILITIES.md:41]: How do you think about pointing to https://github.com/opensearch-project/.github/blob/main/ADMINS.md?

**Comment by dblock** [comment]
> > It is better for members of OpenSearch-Project and worse for the broader community. I would rather be consistent than flaky. Happy to merge with a link target that anyone can follow.
> 
> @peternied Thanks, when could we expected the link mentioned be ready to incorporate here.

Can you PR that change please?

**Comment by seraphjiang** [comment]
> Can you PR that change please?

@dblock @peternied 

currently, the description of admin.md doc limited to this repo only. just confirm the direction moving to have both repo admin and org admin. that means we will maintain two list of admin. admin list for each repo and admin list of org. if confirmed, i'll work on detail update on admin.md in same PR.

> This document explains who the admins are (see below), what they do in this repo, and how they should be doing it.

**Comment by peternied** [comment]
> that means we will maintain two list of admin. admin list for each repo and admin list of org. if confirmed, i'll work on detail update on admin.md in same PR.

👍 Thanks for taking this on @seraphjiang

**Comment by dblock** [comment]
> > Can you PR that change please?
> 
> @dblock @peternied
> 
> currently, the description of admin.md doc limited to this repo only. just confirm the direction moving to have both repo admin and org admin. that means we will maintain two list of admin. admin list for each repo and admin list of org. if confirmed, i'll work on detail update on admin.md in same PR.
> 
> > This document explains who the admins are (see below), what they do in this repo, and how they should be doing it.

For now we don't have a concept of "repo admin" so lets take these things one-by-one, start with whatever is not controversial.

**Comment by seraphjiang** [comment]
> > > Can you PR that change please?
> > 
> > 
> > @dblock @peternied
> > currently, the description of admin.md doc limited to this repo only. just confirm the direction moving to have both repo admin and org admin. that means we will maintain two list of admin. admin list for each repo and admin list of org. if confirmed, i'll work on detail update on admin.md in same PR.
> > > This document explains who the admins are (see below), what they do in this repo, and how they should be doing it.
> 
> For now we don't have a concept of "repo admin" so lets take these things one-by-one, start with whatever is not controversial.

updated, please take a look

File: ADMINS.md
Comment (peternied) [inline ADMINS.md:31]: I prefer responsibilities, how about we keep the text the same?

File: ADMINS.md
Comment (peternied) [inline ADMINS.md:14]: Nit; if you wanted to put a link to the opensearch-project team here, this would be a good spot for it.

"Admin are managed through the OpenSearch-Project organization on team named Admin, members of the organization can see the admin list at [link]"

File: ADMINS.md
Comment (seraphjiang) [reply ADMINS.md:31]: since we have RESPONSIBILITIES.md  to cover, it is the single source of truth.

File: ADMINS.md
Comment (seraphjiang) [reply ADMINS.md:14]: i agree, if the link is not public accessible, it will cause confusion. I hope there is more open and transparent and elegant way to handle this with github.

File: ADMINS.md
Comment (dblock) [inline ADMINS.md:11]: "opensearch project" -> either "OpenSearch Project" or "opensearch-project"

File: ADMINS.md
Comment (dblock) [inline ADMINS.md:29]: Permission -> Permissions

**Review by dblock — CHANGES_REQUESTED** [review]
Looks good, have some language nits.

- update TOC in ADMINS.md
- say that there exists an "admin" team that is added with Admin-level permissions to every repo in the org?

File: ADMINS.md
Comment (seraphjiang) [reply ADMINS.md]: updated ,please take a look

**Review by dblock — APPROVED** [review]
Looks good, but you need to amend your PR to fix DCO, please.

**Review by peternied — APPROVED** [review]
Thanks for the updates, happy to merge after the checks are passing.

**Comment by seraphjiang** [comment]
> Looks good, but you need to amend your PR to fix DCO, please.

fixed dco

**Comment by dblock** [comment]
Thanks! Will merge after built-in bake time. 🍿

**Comment by seraphjiang** [comment]
> Thanks! Will merge after built-in bake time. 🍿

are we good to merge :)

**Pull request merged**
